AutoDraft/sportsradar-hockey.py

```diff
@@ -0,0 +1,68 @@
+#%%[markdown]
```

Comment:
can you add a comment at the top of this file to explain what's going on in the script? I can't tell, but it seems incomplete?

Reply:
Added comment to indicate the package is not developed yet, and its current implementation (ie. VSCode's iPython).